build/root/usr/local/bin/ci_entrypoint_sro.sh

```diff
@@ -5,18 +5,18 @@ set -o errexit
 set -o nounset
 
 prepare_cluster_for_sro() {
-    toolbox/cluster/capture_environment.sh
+    ./run.py cluster capture_environment
     entitle.sh
 
-    if ! toolbox/nfd/has_nfd_labels.sh; then
-        toolbox/nfd-operator/deploy_from_operatorhub.sh
+    if ! ./run.py nfd has_labels; then
```

Comment:
I know this is "inhereted code", but `cmd1 || cmd2` would read a bit better, IMO.

Reply:
I'm personally against using short-circuiting to achieve conditionals, although I see the appeal since it's much shorter

Reply:
I personally really prefer the current way (it was written by myself, but still ^^),
the reason it that it reads very close to the "algorithmic" version"
```
if (cluster does) not have nfd labels,
then deploy nfd-operator from operatorhub
```